code/modules/paperwork/ticketmachine.dm

```diff
@@ -202,3 +225,8 @@
 /obj/item/paper/extinguish()
 	..()
 	update_icon()
+
+/obj/item/ticket_machine_ticket/Destroy()
+	if(owner && source)
+		source.ticket_holders -= owner
+	..()
```

Comment:
``return ..()``
also set owner and source to null and remove src from source.tickets

Reply:
Cobby told me to take the nulls in an earlier review but I'll add them back. Where is the src you're talking about?

Reply:
@nemvar works good now